modules/ROOT/pages/datasense-concept.adoc

```diff
@@ -4,18 +4,33 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, datasense, metadata, meta data, query metadata, dsql, data sense query language
 
-DataSense is a feature of Anypoint Studio. It uses the Mule Event metadata to facilitate application design. Rather than forcing you to manually discover information about the data your internal and external resources employ, Anypoint Studio proactively acquires Mule Event's information such as data type and structure, in order to prescribe how to accurately map or use this data in your application.
+DataSense is a feature of Anypoint Studio that uses Mule event metadata to help you design applications. Rather than forcing you to manually discover this metadata, Anypoint Studio automatically acquires it so that you can map or use this data in your application.
 
-For example, if your application connects to Twitter, DataSense captures information about the type and structure of data that Twitter employs, then uses the information to advise you of the type of data you should send to, or expect to receive from Twitter.
+== Understanding How DataSense Uses Mule Event Metadata
 
-Studio offers these DataSense features to support proactive discovery of metadata from external resources:
+Understanding how your components use metadata in a flow is key to ensuring data reliability by enabling early detection of errors during project design and data reusability in your Mule application by detecting when a component requires metadata that is not being exposed to it within its scope.
```

Comment:
This is one long sentence, can it be broken down? Maybe something like:
...is key to ensuring:
* data reliability, by enabling...
* data reusability, by detecting...